websites/M/Maliki/presence.ts

```diff
@@ -0,0 +1,110 @@
+const presence = new Presence({
+		clientId: "990204576189808670",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+interface songPlayingInformations {
+	artist: string;
+	song: string;
+}
+
+let songInfo: songPlayingInformations = null;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey:
+				document.location.hostname === "malimode.maliki.com"
+					? "malimode"
+					: "maliki_1024",
+			startTimestamp: browsingTimestamp,
+			buttons: [{ label: "Voir le site", url: "https://maliki.com/" }],
+		},
+		//* Get options of the presence
+		[privacy, time] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("time"),
+		]);
+	if (!time) delete presenceData.startTimestamp; // delete the startTimestamp if time option desactivated
```

Comment:
we don't need a comment on that one

Reply:
Corrected in 478d0d23d9b6c22f270fba74fa3b89e24676d745